snap/quota/resources.go

```diff
@@ -53,12 +64,35 @@ func (qr *Resources) validateMemoryQuota() error {
 	return nil
 }
 
+func (qr *Resources) validateCpuQuota() error {
+	// if cpu count is non-zero, then percentage should be set
+	if qr.CPU.Count != 0 && qr.CPU.Percentage == 0 {
+		return fmt.Errorf("cannot validate quota limits with count of >0 and percentage of 0")
```

Comment:
we should make this errors a bit more consistent:

"invalid cpu quota with count ..."

Reply:
done